src/GOVERNANCE.md

```diff
@@ -0,0 +1,108 @@
+# Auro Governance Working Agreement
+
+The Auro Governance Working Agreement is mainly an agreed-upon process for when and how new assets will be added to the Auro Design System.
+
+## New work discovered
+
+Within the scope of a Product Team, new work is created. Engineers and Product designers will review the specification of the feature and determine where new work must happen.
+
+In regards to design, the Product Designer will look to the Auro Design System for opportunities of use.
+
+## Does a component exist?
+
+Here the Product Team is looking to the Auro Design System for assets to use. And per each UI component, they will ask, "*Does this component exist?*"
```

Comment:
"Here..." ... Where? Are you trying to illustrate a situation that this occurs?

Reconsider the way this sentence is delivered to be less narrative.